### DISCUSSION TITLE
Design.

### DISCUSSION THREAD

**Discussion by Gustavekkj** [SuperTux/supertux#2270] [Ideas]
Hi Supertux team
I think supertux need some design rules. Like a color scheme. A smaller pixel density like 16x16 or a way to customize this for a custom sprite sheets. This would make it easier to make design changes, and make the the design of new assets look coherent. If the sprite sheets was smaller and put on a in a single file the workflow would be faster, and allow people to make custom style sheets for other people to download. (Like themes for minecraft). And update to this and the fonts and interface would make this game even better. I don't know if this has any interest, but i am want to help because this would be a big task.

![Screenshot from 2022-07-28 22-53-28](https://user-images.githubusercontent.com/31554493/181636333-7f9f4bb8-a6be-44d4-9307-4389329f1668.png)

![supertux](https://user-images.githubusercontent.com/31554493/181635907-a559a9d6-dd25-4626-9438-9800f0651f54.png)

![super tux](https://user-images.githubusercontent.com/31554493/181636074-3c6ed281-9fd5-423a-b00f-1981f951d8d3.png)

![minitux](https://user-images.githubusercontent.com/31554493/181636146-fdce9f4b-ec5e-4e27-a47c-6b7b4d5258fd.gif)

Sorry english is not my first language.

**Comment by CPa129** [comment]
Um.. These tiles don't fit for the main game, you may use these tiles.

**Comment by IniKiwi** [comment]
i love the texture pack system!
